data/common.yaml

```diff
@@ -0,0 +1,2 @@
+---
+puppet_webhook::pkg_provider: 'puppet_gem'
```

Comment:
please add the missing newline

Reply:
Nit: now there's an empty line

Reply:
This should be fixed now

Reply:
Build is still failing. Did you push the fix?